.github/workflows/_build.yml

```diff
@@ -0,0 +1,272 @@
+# **what?**
+# Build release artifacts and store them to S3 bucket if they do not already exist.
+#
+# Build artifacts get stored in S3 to a bucket with the following directory structure:
+#       "s3://<s3_bucket>/<org>/<repo>/<version>/<commit>/"
+#   ex: "s3://core-team-artifacts/dbt-labs/dbt-core/1.2.3/ce98e6f067d9fa63a9b213bf99ebaf0c29d2b7eb/"
```

Comment:
My concern there is if we try to release with a different commit.  Since the build is essentially cached, we could try to retrigger a release on a new commit and would miss it.  By passing commit here we ensure what we're releasing is exactly what we're expecting.